src/Symfony/Component/Config/Tests/Definition/Dumper/YamlReferenceDumperTest.php

```diff
@@ -69,6 +71,35 @@ public function provideDumpAtPath()
         );
     }
 
+    public function testInvalidYamlExample()
+    {
+        $config = $this->getMockBuilder(ConfigurationInterface::class)->getMock();
```

Comment:
I guess the bug would need to be fixed in 2.7 too, right? Then we cannot use the `class` constant because of our need to still support PHP 5.3 in Symfony 2.x.